pyquil/api/qpu.py

```diff
@@ -60,13 +76,13 @@ class QPUConnection(object):
     Represents a connection to the QPU (Quantum Processing Unit)
     """
 
-    def __init__(self, device_name=None, async_endpoint='https://job.rigetti.com/beta', api_key=None, user_id=None,
-                 ping_time=0.1, status_time=2):
+    def __init__(self, device=None, async_endpoint='https://job.rigetti.com/beta', api_key=None,
+                 user_id=None, ping_time=0.1, status_time=2, device_name=None):
         """
         Constructor for QPUConnection. Sets up necessary security and picks a device to run on.
 
-        :param str device_name: Name of the device to send programs too, should be one of the devices returned from
```

Comment:
We should still document `device_name` as it is still part of the signature, you can add the "deprecated" statement there.

Reply:
disregard this one.